scripts/certs/make_certs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -e
+
+out_dir="/shared/cert"
+
+script_dir="$(dirname "$0")"
+mkdir -p "$out_dir"
+
+# list all certificates we want, so that we can check if they already exist
+server="/shared/cert/ca.crt /shared/cert/server.crt /shared/cert/server.key"
+client="/shared/cert/client.crt /shared/cert/client.key"
+targets="$client $server"
+
+echo ""
+echo "Checking certificates"
+recreate="false"
+# check if certificates exist
+for target in $targets; do
+    if [ ! -f "$target" ]; then
+        echo "$target is missing"
+        recreate="true"
+        break
+    fi
+done
+
+# only recreate certificates if any certificate is missing
+if [ "$recreate" = "false" ]; then
```

Comment:
```suggestion
if ! "$recreate"; then
```

We know `$recreate` is either `true` or `false`, and these are utilities we can use directly in our test.